test/spec/data/facet_spec.js

```diff
@@ -258,5 +258,24 @@ define(['test/mock', 'lodash'], function(mock, _) {
         });
       });
     });
+    describe('on uiClearFacets', function() {
+      beforeEach(function() {
+        this.component.trigger('config', mock.config);
+        this.component.trigger('data', mock.data);
+      });
+      it('emits a "dataFacets" event with no facets selected', function() {
+        this.component.trigger('uiClearFacets', {"facet" : "commudddnity"});
```

Comment:
The nice thing about this library is that you can do it the same way as the UI would, by triggering an event!  In this particular case, you can check out https://github.com/brightredchilli/finda/blob/ying/issue56/test/spec/data/facet_spec.js#L50 which triggers the `uiFilterFacets` event.